### ISSUE TITLE
[docs] Broken on combo trunk (Trac #1613)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1605]
This is the error I get on combo trunk Ubuntu 14.04 with sphinx 1.4 (recently updated 'cause the napolean extension is now required).

Exception occurred:
  File "/usr/local/lib/python2.7/dist-packages/sphinx/domains/cpp.py", line 4039, in _resolve_xref_inner
    assert parentSymbol  # should be there
AssertionError

Migrated from https://code.icecube.wisc.edu/ticket/1613
```json
{
    "status": "closed", 
    "changetime": "2016-04-05T23:31:03", 
    "description": "This is the error I get on combo trunk Ubuntu 14.04 with sphinx 1.4 (recently updated 'cause the napolean extension is now required).\n\nException occurred:\n  File \"/usr/local/lib/python2.7/dist-packages/sphinx/domains/cpp.py\", line 4039, in _resolve_xref_inner\n    assert parentSymbol  # should be there\nAssertionError\n", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "worksforme", 
    "_ts": "1459899063339623", 
    "component": "cmake", 
    "summary": "[docs] Broken on combo trunk", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-03-30T13:43:09", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160330T13:44:07: 
* olivas changed attachment from "" to "sphinx-err-7bVhg_.log"
* olivas commented:

> log file with full traceback

**Comment by icecube-trac** [comment]
Trac update at 20160330T14:42:31: nega commented:

> i just did this a day or two ago and things were fine. i'll double check my versions

**Comment by icecube-trac** [comment]
Trac update at 20160331T20:47:18: 
* nega changed _comment0 from:

> Just tried a clean checkout of combo and I get the same crash.
> 
> writing output... [ 43%] projects/gulliver/seeders
> ('Target: ', u'I3EventHypothesis')
> ('ParentKey: ', [(<ASTNestedNameElement I3SeedServiceBase>, None), (<ASTNestedNameElement I3SeedServiceBase>, None), (<ASTNestedNameElement GetDummy>, None)])
> 
> Exception occurred:
>   File "/usr/local/lib/python2.7/dist-packages/sphinx/domains/cpp.py", line 4039, in _resolve_xref_inner
>     assert parentSymbol  # should be there
> AssertionError
> The full traceback has been saved in /tmp/sphinx-err-w0pRXV.log, if you want to report the issue to the developers.
> Please also report this if it was a user error, so that a better error message can be provided next time.
> A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
> make[3]: *** [cmake/meta-project-docs/CMakeFiles/html] Error 1
> make[2]: *** [cmake/meta-project-docs/CMakeFiles/html.dir/all] Error 2
> make[1]: *** [CMakeFiles/docs.dir/rule] Error 2
> make: *** [docs] Error 2
> 
> Looks like the problem is somewhere in gulliver.
> 

to:

> 1459457888863536


* olivas commented:

> Just tried a clean checkout of combo and I get the same crash.
> 
> 
> ```text
> writing output... [ 43%] projects/gulliver/seeders
> ('Target: ', u'I3EventHypothesis')
> ('ParentKey: ', [(<ASTNestedNameElement I3SeedServiceBase>, None), (<ASTNestedNameElement I3SeedServiceBase>, None), (<ASTNestedNameElement GetDummy>, None)])
> 
> Exception occurred:
>   File "/usr/local/lib/python2.7/dist-packages/sphinx/domains/cpp.py", line 4039, in _resolve_xref_inner
>     assert parentSymbol  # should be there
> AssertionError
> The full traceback has been saved in /tmp/sphinx-err-w0pRXV.log, if you want to report the issue to the developers.
> Please also report this if it was a user error, so that a better error message can be provided next time.
> A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
> make[3]: *** [cmake/meta-project-docs/CMakeFiles/html] Error 1
> make[2]: *** [cmake/meta-project-docs/CMakeFiles/html.dir/all] Error 2
> make[1]: *** [CMakeFiles/docs.dir/rule] Error 2
> make: *** [docs] Error 2
> ```
> 
> Looks like the problem is somewhere in gulliver.
>

**Comment by icecube-trac** [comment]
Trac update at 20160405T23:31:03: 
* olivas changed priority from "critical" to "major"
* olivas changed resolution from "" to "worksforme"
* olivas commented:

> Builds with sphinx 1.3.6.  Still don't know what the issue is with 1.4.
> 
> Not sure whether we want to leave this open or not.  Maybe this is something the sphinx team will magically fix by the time we all *have* to use 1.4 or later.


* olivas changed status from "new" to "closed"
